ui-v2/tests/acceptance/components/catalog-filter.feature

```diff
@@ -60,7 +60,7 @@ Feature: components / catalog-filter
   Where:
     -------------------------------------------------
     | Model   | Page     | Url                       |
-    | service | services | /dc-1/services            |
+    # | service | services | /dc-1/services            |
```

Comment:
Is this intentional? It seems like this is testing the old UX? Perhaps this should just be removed? (same with the ignore statement).

Reply:
Ah actually the other test is _just_ the component not an acceptance test, so seems like we're just missing update this for phrase editing?

Reply:
Ah yeah, the comment should go, will update!

The `@ignore` will be there to pester me until I add a real test for it, it means I get a skipped message whilst running tests, although we are probably at a good point to get that in. Will shout when it's there 👍